File: Hue.Common/CommissionFilterOptions.cs
namespace Hue.Common {
    public class CommissionFilterOptions {
        public int? Page { get; set; } = 0;
        public int? ArtistId { get; set; }
        public int? CommissionTagId { get; set; }
        public int? CharacterId { get; set; }
        public CommissionStatus? CommissionStatus { get; set; }
        public int? Year { get; set; }
    }
}

File: Hue.Web/src/model/commission/CommissionFilterOptions.ts
export default class CommissionFilterOptions {

    //This is NOT a mistake, here we Capitalize the key because the keys still need to be capital for the backend to be on the query

    [key: string]: any

    public Page: number = 0;
    public ArtistId?: number;
    public CommissionTagId?: number;
    public CharacterId?: number;
    public CommissionStatus?: number;
    public Year?: number;

    public Query?: string;

}